src/Desktop/UIAutomation/DesktopElementExtensionMethods.cs

```diff
@@ -145,14 +145,14 @@ private static void AddLogicalSizePseudoProperty(this A11yElement element)
             Rectangle boundingRectangle = element.BoundingRectangle;
             if (boundingRectangle != null && !boundingRectangle.IsEmpty)
             {
-                Point logicalPoint = new Point(
+                Point boundingRectangleCenter = new Point(
                     (boundingRectangle.Left + boundingRectangle.Right) / 2, 
                     (boundingRectangle.Top + boundingRectangle.Bottom) / 2);
-                Win32Helper.GetDpi(logicalPoint, DpiType.Effective, out uint rawDPIX, out uint RawDPIY);
+                Win32Helper.GetDpi(boundingRectangleCenter, DpiType.Effective, out uint rawDPIX, out uint rawDPIY);
                 double xScale = rawDPIX / defaultDpi;
-                double yScale = RawDPIY / defaultDpi;
-                logicalSize[0] = (int)(boundingRectangle.Width * xScale);
-                logicalSize[1] = (int)(boundingRectangle.Height * yScale);
+                double yScale = rawDPIY / defaultDpi;
+                logicalSize[0] = (int)(boundingRectangle.Width / xScale);
```

Comment:
I assume this is the main "fix-up" in this PR (the change from multiplication to division). Adding a link to the comment in the previous PR for posterity: https://github.com/microsoft/axe-windows/pull/1020/files#r1602381757